app/views/posts/show.html.erb

```diff
@@ -1,2 +1,40 @@
-<h1>Specific Post of a User</h1>
-<p>Specific post with post id will appear here</p>
+<h1>Selected Post of a User</h1>
+
+<main id="post-index" class="page">
+<div id="user-page">
+ <div class="btn-container">
+      <%= link_to "Home", root_path %>
+
+
```

Comment:
![image](https://github.com/baqar-abbas/blog-app-rails/assets/93039359/32fce6a4-203d-4344-a664-bdca8fe4ef52)

- [ ] It would be nice if you fix the highlighted link in the image above ⬆️ because right now it is broken 😨 Kindly fix it or remove it (If you don't have any dedicated section for the highlighted link) to make your readme descriptive and professional 👍

Reply:
Deployment link has been removed from the read me file. Screen shot attached below for your kind reference.
![image](https://github.com/baqar-abbas/blog-app-rails/assets/117091923/a3c089fb-b4d8-4709-86bc-b200a5dcfee6)